### PR TITLE
[Remove Stream/Optional usage] LPS-176103 Remove_Stream_usages_for_module_calendar_test

### PR DESCRIPTION
**Pull request by jeffwu0724**
Hi Tina, Dante,

Resent: https://github.com/liferay-core-infra/liferay-portal/pull/3557
Thanks for checking.

### CODE REVIEW

File: modules/apps/calendar/calendar-test/src/testIntegration/java/com/liferay/calendar/search/test/CalendarBookingIndexerLocalizedContentTest.java [hunk 1]
```diff
@@ -89,32 +89,18 @@ public void testJapaneseTitle() throws Exception {
...
-		String prefix1 = "新";
-		String prefix2 = "作";
-
-		Stream.of(
-			word1, word2, prefix1, prefix2
-		).forEach(
-			keywords -> assertFieldValues(
-				"title", LocaleUtil.JAPAN, titleMap, keywords)
-		);
+		for (String keyword : Arrays.asList("新規", "作成", "新", "作")) {
```
Comment (jeffwu0724) [inline modules/apps/calendar/calendar-test/src/testIntegration/java/com/liferay/calendar/search/test/CalendarBookingIndexerLocalizedContentTest.java:92]: use Arrays.asList() directly while iterating. 
Also inline the word without declaring new variables

**Comment by Kvale1733** [comment]
ci:test:calendar

**Comment by Kvale1733** [comment]
@jeffwu0724 No regressions found.

**Comment by dantewang** [comment]
ci:test:force

cc @shuyangzhou @tinatian

**Comment by dantewang** [comment]
ci:forward:force

**Pull request closed without merging**